test/cli-validator/tests/expectedOutput.js

```diff
@@ -87,17 +87,20 @@ describe('cli tool - test expected output - Swagger 2', function() {
 
     // .match(/\S+/g) returns an array of all non-whitespace strings
     //   example output would be [ 'Line', ':', '59' ]
+    //errors
```

Comment:
small formatting suggestion

```suggestion

    // errors
```